flambe/field/text.py

```diff
@@ -295,7 +319,8 @@ def setup(self, *data: np.ndarray) -> None:
             self.vocab, self.embedding_matrix = self._build_embeddings(self.model)
 
     # TODO update when we add generics
-    def process(self, example: str) -> torch.Tensor:  # type: ignore
+    def process(self, example: Union[str, List[Any], Dict[Any, Any]]) \
```

Comment:
docstring isn't quite right. Should be  `Union[str, List[str], Dict[Any, str]]`

Reply:
What if you have nested objects? E.g., `{'asdf': ['asdf', 'asdf'], {'asdf': {'asdf': ['asdf', 'asdf'], 'a2': 'asdf'}}}`

Reply:
Does the PR support that?

Reply:
yes, completely. There are even testcases for this.